src/benchmark/cases/quicksort.js

```diff
@@ -48,4 +48,54 @@ function quickSort(items, left, right) {
     return items;
 }
 
-module.exports = {quickSort, partition, swap};
+// LCG with GCCs parameters
+function fill(array) {
+    var i = 0;
+
+    var modulus = 0x80000000; // 2^31
+    var multiplier = 1103515245;
+    var increment = 12345;
+
+    var state = 42;
+
+    while (i < array.length) {
+        state = (multiplier * state + increment) % modulus;
```

Comment:
The multiplication creates an integer overflow, resulting in bad wasm times and wrong results.

Reply:
Yes there are integer overflows because of the multiplication but the wasm time is not bad at all and the result is still correct. Maybe there is a PRNG that works with 32-bit integers

Reply:
If the wasm time isn't bad because of the overflows and the result is correct then I don't see the need to change it.